tower_cli/resources/job_template.py

```diff
@@ -139,7 +139,11 @@ def associate_credential(self, job_template, credential):
 
         =====API DOCS=====
         """
-        return self._assoc('extra_credentials', job_template, credential)
+        try:
+            # Pattern in 3.3
+            return self._assoc('credentials', job_template, credential)
```

Comment:
might want `exc. NotFound` for the exception, also might want to log this.


But we almost might prefer a new exception type for #442